src/apps/miscapps/generateBaseCellNeighbors.c

```diff
@@ -237,24 +223,25 @@ void generate() {
 
     printf("const int baseCellNeighbors[NUM_BASE_CELLS][7] = {\n");
     for (int i = 0; i < NUM_BASE_CELLS; i++) {
-        char* neighborStrings[7] = {0, 0, 0, 0, 0, 0, 0};
+        printf("{");
         for (int j = 0; j < 7; j++) {
-            neighborStrings[j] = getBaseCellOutput(baseCellNeighbors[i][j]);
+            if (j > 0) {
+                printf(", ");
+            }
+            if (baseCellNeighbors[i][j] != INVALID_BASE_CELL) {
+                printf("%d", baseCellNeighbors[i][j]);
+            } else {
+                printf("INVALID_BASE_CELL");
+            }
         }
-        printf("{%s, %s, %s, %s, %s, %s, %s}, // base cell %d%s\n",
-               neighborStrings[0], neighborStrings[1], neighborStrings[2],
-               neighborStrings[3], neighborStrings[4], neighborStrings[5],
-               neighborStrings[6], i,
+        printf("}, /* base cell %d%s */\n", i,
```

Comment:
Since the formatting of the comments changes, could you rerun generateBaseCellNeighbors and update baseCells.c?

Reply:
Sure thing.